docs/docs/reference/sandbox_reference/sandbox-reference.md

```diff
@@ -155,7 +156,7 @@ The P2P Bootstrap node is a standalone app whose purpose is to assist new P2P ne
 P2P_TCP_LISTEN_IP='0.0.0.0' # The IP Address on which to listen for connections.
 P2P_TCP_LISTEN_PORT=40400 # The port on which to listen for connections.
 PEER_ID_PRIVATE_KEY='' # The private key to be used by the peer for secure communications with other peers. This key will also be used to derive the Peer ID.
-P2P_ANNOUNCE_HOSTNAME='' # The IPAddress/Hostname that other peers should use to connect to this node, this may be different to P2P_TCP_LISTEN_IP if e.g. the node is behind a NAT.
+P2P_ANNOUNCE_UDP_HOSTNAME='' # The IPAddress/Hostname that other peers should use to connect to this node, this may be different to P2P_TCP_LISTEN_IP if e.g. the node is behind a NAT.
```

Comment:
I think this env var is a duplicate of the one on line 120

Reply:
separate docs for starting P2P Bootstrap node, rather than an aztec node